src/app/chip_data_model.gni

```diff
@@ -105,6 +105,7 @@ template("chip_data_model") {
       "${_app_root}/util/ember-compatibility-functions.cpp",
       "${_app_root}/util/ember-print.cpp",
       "${_app_root}/util/error-mapping.cpp",
+      "${_app_root}/util/generic-callback-stubs.cpp",
```

Comment:
You need to also update `chip_data_model.cmake`.

Reply:
You are right - fixed it and CI should be green now.

I also see maybe src/darwin/Framework/Matter.xcodeproj would need those, but I do not know how to update them :/ CI seems happy even without that though.

@bzbarsky-apple